server/src/main/java/org/apache/iotdb/db/mpp/aggregation/slidingwindow/SlidingWindowAggregator.java

```diff
@@ -54,7 +54,7 @@ protected SlidingWindowAggregator(
   }
 
   @Override
```

Comment:
consider not letting SlidingWindowAggregator extend Aggregator, maybe the code will be clearer

Reply:
If SlidingWindowAggregator doesn't extends Aggregator, there will be more common codes appear in SlidingWindowAggregator (and SlidingWindowAggregationOperator needs to be modified, too). So I just choose to delete the sliding-window-method in Aggregator and add a type cast in SlidingWindowAggregationOperator now.